curvefs/conf/client.conf

```diff
@@ -88,6 +88,11 @@ fuseClient.dCacheLruSize=65536
 fuseClient.enableICacheMetrics=true
 fuseClient.enableDCacheMetrics=true
 fuseClient.cto=true
+# splice will bring higher performance in some cases
+# but there're might be a kernel issue that will cause kernel panic when enabling it
+# see https://lore.kernel.org/all/CAAmZXrsGg2xsP1CK+cbuEMumtrqdvD-NKnWzhNcvn71RV3c1yw@mail.gmail.com/
+# until this issue has been fixed, splice should be disabled
+fuseClient.enableSplice=false
```

Comment:
Note that there is no problem starting from 5.10, there is a problem with 5.4 and before, and the version between 5.4 and 5.10 is uncertain

Reply:
yep, but hard code a kernel version is not flexible, because someone may patch fixes and then splice can be enabled.

Reply:
OK